modules/aws-in-a-box/metadata.json

```diff
@@ -22,7 +22,8 @@
         "0.0.58",
         "0.0.61",
         "0.0.62",
-        "0.0.63"
+        "0.0.63",
+        "0.0.65"
```

Comment:
Please resolve the conflict caused by https://github.com/bazelbuild/bazel-central-registry/pull/2809

Reply:
done